knexfile.js

```diff
@@ -23,7 +24,12 @@ module.exports = {
   },
   testing: {
     ...dbObj,
-    connection: process.env.TESTING_DB_URL
+    connection: process.env.TESTING_DB_URL,
+    pool: {
+      min: 0,
+      max: 10,
+      idleTimeoutMillis: 500
```

Comment:
verbally communicated with Cody about documenting the process or any harder to solve errors he came across for testing with a postgres DB.

Reply:
Brianna makes a great point about documenting errors and troubleshooting for future project members, will add to documentation.